core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -660,8 +660,18 @@ class HttpGoogleServicesDAO(
       //Iterate over each chunk.
       val allProcessedChunks: IO[List[Seq[RawlsBillingAccount]]] = accountChunks traverse { chunk =>
 
+        //Filter out the billing accounts that are closed. They have no value to users
+        //and can cause confusion by cluttering their lists
```

Comment:
I agree that there isn't much benefit to showing closed billing accounts in the UI. How did you decide to make the change in the API vs the UI?

Reply:
On the off-chance that this info may be useful to some users, maybe it's worth getting a PO to sign off on this change?

Reply:
The feedback I got from the Frontline Support team and the SUE team's PO was that these won't ever have value to users. With that feedback, it felt better (and truthfully, easier) to filter them out at the API level rather than changing the API response signature or adding query params in order to do it in the UI

Reply:
awesome, thank you!